src/coreComponents/linearAlgebra/unitTests/testVectors.cpp

```diff
@@ -150,7 +150,7 @@ TYPED_TEST_P( VectorTest, copyConstruction )
   using Vector = typename TypeParam::ParallelVector;
 
   Vector x;
-  createAndAssemble< parallelDevicePolicy<> >( 3, x );
+  createAndAssemble< geos::parallelDevicePolicy<> >( 3, x );
```

Comment:
maybe just add a `using namespace geos` at the top of this source?

Reply:
Unfortunately the unit test compile time is pushing our worst case CI back up to 3 hours, it is on its 3rd attempt to pass.

Reply:
I would suggest adding `--build-exe-only` here: https://github.com/GEOS-DEV/GEOS/blob/f12d9c93cf1bc1c7cfc243ac42067a498c387171/.travis.yml#L179 

coz even if it finally passes, all following PRs will battle with this.

Reply:
we can turn them back on as soon as we move to `githubactions`.